### PR TITLE
DAS-1885: Mid-flight update to regression tests.

### PR DESCRIPTION
**Pull request by flamingbear**
This only makes them pass.

## Description

Update regression tests to pass CI/CD

## Jira Issue ID

DAS-1943

## Local Test Steps

Run this locally by checking out this branch, changing harmony_host_url to `http:localhost:3000/` and running the notebook or by setting
it to harmony_host_url = 'https://harmony.sit.earthdata.nasa.gov' and running against sit.

## PR Acceptance Checklist
* [X] Acceptance criteria met
* [n/a] Tests added/updated (if needed) and passing
* [n/a] Documentation updated (if needed)

### CODE REVIEW

File: test/hybig/reference_data/ASTGTMV003_N00E022_dem.jpg
Comment (flamingbear) [inline test/hybig/reference_data/ASTGTMV003_N00E022_dem.jpg:1]: That little speck of dust is the output area in the new image.

File: test/hybig/reference_data/ASTGTMV003_N00E022_dem.png
Comment (flamingbear) [inline test/hybig/reference_data/ASTGTMV003_N00E022_dem.png:1]: Same speck of dust, but on a transparent png image.

**Review by flamingbear — COMMENTED** [review]
just leaving notes.

**Review by owenlittlejohns — APPROVED** [review]
I built the new Docker image (v0.0.2) locally and ran the test suite against SIT. Everything ran as expected.

I also took a look at the teeny-tiny images to ensure that they still looked like the correct region. Within the limitations of high-pixelation, the images looked right to me. (Probably points to the collection being a good candidate for tests where the Harmony message defines non-GIBS-compatible output during DAS-1871).

**Pull request merged**